cirq-core/cirq/work/observable_settings.py

```diff
@@ -14,6 +14,7 @@
 
 import dataclasses
 from typing import Union, Iterable, Dict, TYPE_CHECKING, ItemsView, Tuple, FrozenSet
+import sympy
```

Comment:
nit: separate from std lib imports
```suggestion

import sympy
```